test/rules/import-blacklist-submodules/test.ts.lint

```diff
@@ -0,0 +1,25 @@
+import { submodule } from '@blueprintjs/core';
```

Comment:
nit: please use double quotes to be consistent with the default config presets (`quotemark: [true, "double"]`)